### DISCUSSION TITLE
Trouble insalling "birdnetR"

### DISCUSSION THREAD

**Discussion by leahmctigue** [birdnet-team/birdnetR#22] [Q&A]
Hey all - I'm having alot of trouble downloading birdnetR. I continually get the following error: 

Error: package or namespace load failed for 'birdnetR':
 .onLoad failed in loadNamespace() for 'birdnetR', details:
  call: value[[3L]](cond)
  error: unused argument (cond)
Error: loading failed
Execution halted
ERROR: loading failed
* removing 'C:/Users/LeahM/AppData/Local/R/win-library/4.4/birdnetR'
Warning: installation of package ‘C:/Users/LeahM/AppData/Local/Temp/RtmpYT27eM/file3098616f15b/birdnetR_0.2.0.tar.gz’ had non-zero exit status

I have updated R, and all packages, and have restarted this process several times. Is there something I'm missing? I have been following the R markdown vignette. 

Thanks!

**Comment by fegue** [comment]
Thanks for reporting your issue! 
Since you are using Windows, this could be a related problem.
Does this happen when you run `devtools::install_github("birdnet-team/birdnetR")`?

**Comment by leahmctigue** [comment]
Hi! Thank you for the response. Yes, this is happening on that line of code.

Here are some steps I have already tired:

  *   Removing devtools and reinstalling with the most updated version
  *   Downloading R with the most recent version
  *   Running the code while both in and out of the birnetR project
  *   Ensuring all installed packages were stored in the same location on my PC
  *   Ensuring all packages are up to date (devtools was having trouble updating, hence the removal and reinstall)

I appreciate any help you can provide!

Leah

**Reply by BaileyKylie** [reply]
I also have been having problems with this!!! Have been trying to troubleshoot all day !

**Comment by fegue** [comment]
Unfortunately, I cannot reproduce the issue.
Have you tried installing the new version recently?
Hopefully, the issue will be resolved, when we have the package on CRAN later this year.

**Comment by fegue** [comment]
@leahmctigue @BaileyKylie the package is now available from CRAN and uses a different approach to manage the Python environment. 
Please try to install it with `⁣install.packages("birdnetR")` and check if it works for you.
Thanks!
